yt_dlp/extractor/mixch.py

```diff
@@ -62,20 +69,44 @@ class MixchArchiveIE(InfoExtractor):
             'id': '421',
             'title': '96NEKO SHOW TIME',
         }
+    }, {
+        'url': 'https://mixch.tv/archive/1213',
+        'skip': 'paid video, no DRM. expires at Dec 31, 2023',
+        'info_dict': {
+            'id': '1213',
+            'title': '【特別トーク番組アーカイブス】Merm4id×燐舞曲 2nd LIVE「VERSUS」',
+            'live_status': 'not_live',
+            'release_timestamp': 1701421200,
+        }
+    }, {
+        'url': 'https://mixch.tv/archive/1214',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        webpage = self._download_webpage(url, video_id)
 
-        html5_videos = self._parse_html5_media_entries(
-            url, webpage.replace('video-js', 'video'), video_id, 'hls')
-        if not html5_videos:
+        info_json, urlh = self._download_json_handle(
+            f'https://mixch.tv/api-web/archive/{video_id}', video_id, expected_status=(401))
+        if urlh.status == 401:
             self.raise_login_required(method='cookies')
-        infodict = html5_videos[0]
-        infodict.update({
-            'id': video_id,
-            'title': self._html_search_regex(r'class="archive-title">(.+?)</', webpage, 'title')
-        })
+            return
```

Comment:
Since we raise an error no return is needed
```suggestion
```

Reply:
Yes, we always raise `ExtractorError` if `metadata_available=False` .